services/slam/builtin/orbslam_int_test.go

```diff
@@ -62,7 +62,7 @@ func testPositionAndMap(t *testing.T, svc slam.Service) {
 	// Typical values are around (0.001, -0.001, 0.007)
```

Comment:
I'm curious if this should change too, now that we're taking the inverse of the pose in orbslam before extracting the translation.